azurerm/resource_arm_virtual_machine.go

```diff
@@ -334,7 +334,7 @@ func resourceArmVirtualMachine() *schema.Resource {
 
 						"admin_password": {
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
```

Comment:
Shouldn't we also mark this as `Sensitive` so it doesn't show up in the plan/apply output?